app/views/layouts/_social_media.html.erb

```diff
@@ -0,0 +1,11 @@
+<% SiteConfig.social_media_handles.keys.each do |social_media_type| %>
```

Comment:
We show social media icons in more than one place so it makes sense to pull it out into a partial.